src/api_stackflow.js

```diff
@@ -0,0 +1,31 @@
+//install npm i node-fetch --save
+
+const fetch = require("node-fetch"); 
+
+var method=""
+var library=""
```

Comment:
```suggestion
let method="";
let library="";
```

Reply:
we should use `let` or `const` for variables when we can. `var` is a bit outdated as all variables created with it are global